### PR TITLE
Easy opt-out handle

### PR DESCRIPTION
**Pull request by gsvr**

### CODE REVIEW

**Comment by gsvr** [comment]
@imsickofmaps Please review.

File: controlinterface/views.py [hunk 1]
```diff
@@ -217,6 +218,23 @@ def subscription_edit(request):
                     "confirmbabyform": confirmbabyform,
                 })
                 context.update(csrf(request))
+    elif request.method == "GET" and 'msisdn' in request.GET:
+        # Confirm before updating the record
+        msisdn = request.GET.get("msisdn")
+
```
Comment (imsickofmaps) [inline controlinterface/views.py:235]: We need to validate this msisdn has subs. If not, switch to just an optout button. For consistency please also add this to the current lookup screen.

File: controlinterface/templates/controlinterface/subscription.html [hunk 2]
```diff
@@ -47,29 +47,73 @@ <h1 class="page-header">Subscriptions</h1>
...
+            {% csrf_token %}
+            {{ optoutform|bootstrap }}
+              <input type="submit" class="btn btn-success" value="Full Opt-Out" />
+            </form>
+          </p>
+          <p>
+            <form id="subscription-edit-form" method="post" method="POST" action=".">
+            {% csrf_token %}
+            {{ form|bootstrap }}
+              <input type="submit" class="btn btn-danger" value="No, don't cancel subscriptions" />
```
Comment (imsickofmaps) [inline controlinterface/templates/controlinterface/subscription.html]: `No, don't cancel subscriptions` should be `No, don't fully opt-out`

**Comment by imsickofmaps** [comment]
:+1:

**Pull request merged**